pkg/controllers/jobset_controller.go

```diff
@@ -323,13 +340,55 @@ func (r *JobSetReconciler) createHeadlessSvcIfNotExist(ctx context.Context, js *
 	return nil
 }
 
+func (r *JobSetReconciler) executeSuccessPolicy(ctx context.Context, js *jobset.JobSet, ownedJobs *childJobs) error {
+	switch js.Spec.SuccessPolicy.Operator {
+	case jobset.OperatorAll:
+		return r.successPolicyAll(ctx, js, ownedJobs)
+	case jobset.OperatorAny:
+		return r.successPolicyAny(ctx, js, ownedJobs)
+	default:
+		return fmt.Errorf("invalid operator %s", js.Spec.SuccessPolicy.Operator)
```

Comment:
Should we move this error check into a validating webhook?

Reply:
yes, I added a comment on the api